addons/route53-mapper/v1.3.0.yml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: extensions/v1beta1
+apiVersion: app/v1
```

Comment:
```suggestion
apiVersion: apps/v1
```
I believe it should be plural

See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#deployment-v1-apps

Reply:
Transcription error. My apologies.